### ISSUE TITLE
Jester Hat

### ISSUE THREAD

**Issue by SamJakob** [Templarian/MaterialDesign#6788]
### Before submitting a new icon request, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](http://dev.materialdesignicons.com/roadmap/brand-icons) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I only intend to request a single icon (or a few near-identical ones) in this issue.

### What would you use this icon for?

For playing cards, or for a logo for 'Entertainment' or 'Joke'/'Humor' categories in computer software.

### Give us an example of what this icon might look like.

![image](https://user-images.githubusercontent.com/37072691/192234055-58a80680-5826-482c-9fe0-c6466d568341.png)
![image](https://user-images.githubusercontent.com/37072691/192234157-f7176665-aadd-4c29-a0d5-d11a2df2f5a5.png)
![image](https://user-images.githubusercontent.com/37072691/192234239-fbfcd8c4-2398-42ea-b5c4-1ae1af19ec4d.png)

**Comment by suterma** [comment]
For playing cards, there is literally a "cards" icon, plus several for the typical categories like heart, spade, etc.

Although there really is one lacking for "fun/humor/joke" etc. and I like your proposal for a jester hat.

**Comment by SamJakob** [comment]
Yes all the playing cards are covered except the Jokers/Jesters - hence the request.

But absolutely - it would also serve as a general icon for fun, jokes, humor, etc., and definitely serves a generally useful purpose too.

**Comment by Dragilooping** [comment]
![Image](https://github.com/user-attachments/assets/a0300cb4-cb79-4735-bbab-9f2170e2d907)
[jester-hat.zip](https://github.com/user-attachments/files/19160530/jester-hat.zip)
